### PR TITLE
OBPIH-2636 Update order status when stock movement status changes

### PR DESCRIPTION
**Pull request by jmiranda**

### CODE REVIEW

File: grails-app/domain/org/pih/warehouse/order/Order.groovy [hunk 1]
```diff
@@ -109,6 +112,21 @@ class Order implements Serializable {
         return status
     }
 
+    def getDisplayStatus() {
+        def shipments = listShipments()
```
Comment (awalkowiak) [inline grails-app/domain/org/pih/warehouse/order/Order.groovy]: I just realized that this is already there, I'll rewrite my pr (https://github.com/openboxes/openboxes/pull/1518)

Comment (jmiranda) [reply grails-app/domain/org/pih/warehouse/order/Order.groovy]: @awalkowiak i saw this comment but thought you had made it in one of your PRs. i didn't realize you noticed it in my PR :laughing:

Comment (awalkowiak) [reply grails-app/domain/org/pih/warehouse/order/Order.groovy]: I noticed it here, but I refactored this in my. there will be need for veeery small rebase in this or in my PR after merge (depends on which will be merged first)

**Comment by awalkowiak** [comment]
@jmiranda I think you need to rebase it and change usage of listShipments().

**Pull request merged**